evm/src/fuzz/strategies/invariants.rs

```diff
@@ -84,11 +86,27 @@ fn generate_call(
 
 /// Strategy to select a sender address:
 /// * If `senders` is empty, then it's a completely random address.
```

Comment:
Just to confirm: It's not "completely random" right? But instead is either random OR from the dict, with the same weights as with other fuzz values

Reply:
@joshieDo same Q, what's the difference between `fuzz_param_from_state` that has a 90% chance of being selected in `fuzz_strategy` vs the `senders: Vec<Address>` being passed in the fn?

Reply:
updated the docs to:
```
/// Strategy to select a sender address:
/// * If `senders` is empty, then it's either a random address (10%) or from the dictionary (90%).
/// * If `senders` is not empty, then there's an 80% chance that one from the list is selected. The
///   remaining 20% will either be a random address (10%) or from the dictionary (90%).
```